docs/sphinx/user_guide/feature/tiling.rst

```diff
@@ -117,3 +145,29 @@ parameter '0') and the local tile loop index is the third lambda argument
           arguments. Then, the parameter tuples identified by the integers 
           in the ``RAJA::Param`` statement types given for the loop statement 
           types follow.
+	
+The launch API uses ``RAJA::tile_tcount`` and ``RAJA::loop_icount`` methods
+which has a second argument on the lambda for the index. We illustrate usage below::
+
+  using launch_t = RAJA::LaunchPolicy<RAJA::seq_launch>;
+  using loop_t   = RAJA::LoopPolicy<RAJA::loop_exec>;
+
+  RAJA::launch<launch_t>
+  (RAJA::LaunchParams(), RAJA_HOST_DEVICE(RAJA::launchContext ctx)
+  {
+
+    RAJA::tile_tcount<loop_t>
+      (ctx, tile_size, RAJA::TypedRangeSegment<int>(0, 10), [&] (RAJA::TypedRangeSegment<int> const &tile, int t) {
+
+      RAJA::loop_icount<loop_t>
+        (ctx, tile, [&] (int idx, int i) {
+
+        // idx - global index
+        // t - tile number
+        // i - index within tile
+	// Then, idx = i + tile_size*t
+
+	
+	});      

```

Comment:
vertical alignment of }); and statement it corresponds to

Reply:
Thanks, just pushed up the formatting changes.